scripts/gha/print_matrix_configuration.py

```diff
@@ -127,11 +133,13 @@
   "2017": {
     _WINDOWS: {
       "version": "2017.4.40f1",
-      "packages": {"Default": ["Unity"], "Android": ["Android"], "iOS": ["Ios"], "Windows": None, "macOS": ["Mac"], "Linux": ["Linux"]}
+      "packages": {"Default": ["Unity"], "Android": ["Android"], "iOS": ["Ios"], "Windows": None, "macOS": ["Mac"], "Linux": ["Linux"]},
+      "ndk": ""
```

Comment:
do we want to keep it consistency with _LINUX setup, either also have an empty entry in _LINUX, or we don't need to have this under 2017?

Reply:
I feel both is fine, especially we are sunsetting Unity 2017.
Currently, the code goes like this. (We don't build testapps on Unity machine though.)
```
  if UNITY_SETTINGS[str(major_version)][get_desktop_platform()]["ndk"]:
    url = UNITY_SETTINGS[str(major_version)][get_desktop_platform()]["ndk"]
```